index.js

```diff
@@ -1,30 +1,36 @@
 'use strict'
 
-const holidays = require('./holidays/2017')
+const holidays = require('./holidays/')
+
+const holiday = obj => {
```

Comment:
Also create a test for that, running the function without any arguments should return the entire year